awscli/customizations/s3/utils.py

```diff
@@ -353,6 +353,44 @@ def relative_path(filename, start=os.path.curdir):
         return os.path.abspath(filename)
 
 
+def is_glacier_object(response_data):
```

Comment:
I extracted it because I saw it as being potential useful if you were to write a glacier-specific sync strategy. I am fine with moving it to FileInfo though since we are not using it anywhere else.